src/components/Person/Person/Person.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partner = sex === 'm'
+    ? 'wife'
+    : 'husband';
+
+  const isUserMarried = isMarried
+    ? `${partnerName} is my ${partner}`
+    : `I am not married`;
+
+  return (
+    <>
+      <section className="Person">
+        <h2 className="Person__name">{`My name is ${name}`}</h2>
+        {age && (
+        <p className="Person__age">{`I am ${age}`}</p>
+        )}
+        <p className="Person__partner">{isUserMarried}</p>
```

Comment:
add some blank lines between elements to improve readability
```suggestion
        <h2 className="Person__name">{`My name is ${name}`}</h2>
        
        {age && (
        <p className="Person__age">{`I am ${age}`}</p>
        )}
        
        <p className="Person__partner">{isUserMarried}</p>
```